test_responses.py

```diff
@@ -390,3 +390,14 @@ def run():
 
     run()
     assert_reset()
+
```

Comment:
@bmwant not sure to understand ...

tests actually pass on my laptop, in 2.7

```
(press) ~/src/python/responses [raw]$ py.test
===================================================================== test session starts =====================================================================
platform linux2 -- Python 2.7.11+, pytest-2.9.2, py-1.4.31, pluggy-0.3.1
rootdir: /home/thenrio/src/python/responses, inifile: setup.cfg
plugins: cov-1.7.0
collected 19 items 

test_responses.py ...................

================================================================== 19 passed in 0.13 seconds ==================================================================
```

and end of file is pretty much the same with master

```
(press) ~/src/python/responses [raw]$ git show master:test_responses.py | tail -1 | od -tc
0000000                   a   s   s   e   r   t   _   r   e   s   e   t
0000020   (   )  \n
0000023
(press) ~/src/python/responses [raw]$ git show raw:test_responses.py | tail -1 | od -tc
0000000                   a   s   s   e   r   t   _   r   e   s   e   t
0000020   (   )  \n
0000023
```

so what is your command that does not pass ?